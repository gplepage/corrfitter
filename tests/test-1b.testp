(dp0
S'log(Ds:a)'
p1
cnumpy.core.multiarray
_reconstruct
p2
(cnumpy
ndarray
p3
(I0
tp4
S'b'
p5
tp6
Rp7
(I1
(I1
tp8
cnumpy
dtype
p9
(S'f8'
p10
I0
I1
tp11
Rp12
(I3
S'<'
p13
NNNI-1
I-1
I0
tp14
bI00
S'\xb4\x9d}\x19i\x9b\xf8\xbf'
p15
tp16
bsS'log(Ds:ao)'
p17
g2
(g3
(I0
tp18
g5
tp19
Rp20
(I1
(I1
tp21
g12
I00
S'\xc1\xbc\xd5\xf9\xa6\xc5\x04\xc0'
p22
tp23
bsS'Vno'
p24
g2
(g3
(I0
tp25
g5
tp26
Rp27
(I1
(I1
I1
tp28
g12
I00
S'?\xbb\xcd\x06\x04\xb9\xe7\xbf'
p29
tp30
bsS'Vnn'
p31
g2
(g3
(I0
tp32
g5
tp33
Rp34
(I1
(I1
I1
tp35
g12
I00
S'\x14\xb4&1\xed\x9e\xe8?'
p36
tp37
bsS'log(Ds:dEo)'
p38
g2
(g3
(I0
tp39
g5
tp40
Rp41
(I1
(I1
tp42
g12
I00
S'[4;\xfd~&\xd8?'
p43
tp44
bsS'log(etas:dE)'
p45
g2
(g3
(I0
tp46
g5
tp47
Rp48
(I1
(I1
tp49
g12
I00
S'\xd0"\xbf\x94\x83\x0c\xec\xbf'
p50
tp51
bsS'log(Ds:dE)'
p52
g2
(g3
(I0
tp53
g5
tp54
Rp55
(I1
(I1
tp56
g12
I00
S'T\xbc\xb1\x92\xa9\x85\xc7?'
p57
tp58
bsS'log(etas:a)'
p59
g2
(g3
(I0
tp60
g5
tp61
Rp62
(I1
(I1
tp63
g12
I00
S'\xd6G\x94\x04<W\xf8\xbf'
p64
tp65
bs.