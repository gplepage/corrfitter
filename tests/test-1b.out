========================== nexp = 1   ratio = True
Least Square Fit (input data correlated with prior):
  chi2/dof [dof] = 0.76 [69]    Q = 0.93    logGBF = -74.778    itns = 1

Parameters:
    log(etas:a)_ -1.521 +-   0.001           (    -1.2 +-        1)
   log(etas:dE)_ -0.877 +-   0.000           (   -0.92 +-      0.5)
      log(Ds:a)_ -1.538 +-   0.001           (    -1.2 +-        1)
     log(Ds:dE)_  0.184 +-   0.000           (    0.18 +-     0.17)
     log(Ds:ao)_ -2.597 +-   0.023           (    -2.3 +-        1)
    log(Ds:dEo)_  0.377 +-   0.003           (    0.41 +-     0.24)
            Vnn_  0.769 +-   0.001           (     0.1 +-        1)
            Vno_ -0.741 +-   0.019           (     0.1 +-        1)

etas:dE = 0.41623(8)
 etas:E = 0.41623(8)  0.41623(8)  0.41623(8)
 etas:a = 0.2184(1)

Ds:dE = 1.2017(1)
 Ds:E = 1.2017(1)  1.2017(1)  1.2017(1)
 Ds:a = 0.2148(2)

etas->V->Ds = 0.769(1)



Number of bootstrap iterations = 4 

     label             bs-median               bs-gvar                   fit
----------------------------------------------------------------------------
   etas:dE             0.4162(2)            0.41625(8)            0.41623(8)
    etas:a             0.2184(1)            0.21840(6)             0.2184(1)
     Ds:dE             1.2018(1)            1.20178(9)             1.2017(1)
      Ds:a             0.2149(2)           0.21488(10)             0.2148(2)
       Vnn              0.771(2)             0.7713(8)              0.769(1)
       Vno              -0.73(1)              -0.73(1)              -0.74(2)
